extension.neon

```diff
@@ -146,12 +146,47 @@ services:
 		factory: PHPStan\Type\Symfony\InputInterfaceHasOptionDynamicReturnTypeExtension
 		tags: [phpstan.broker.dynamicMethodReturnTypeExtension]
 
+	# ArrayNodeDefinition::*prototype() return type
+	-
+		factory: PHPStan\Type\Symfony\Config\ArrayNodeDefinitionPrototypeDynamicReturnTypeExtension
+		tags: [phpstan.broker.dynamicMethodReturnTypeExtension]
+
+	# ExprBuilder::end() return type
+	-
+		factory: PHPStan\Type\Symfony\Config\ReturnParentDynamicReturnTypeExtension
+		arguments: ['Symfony\Component\Config\Definition\Builder\ExprBuilder', ['end']]
```

Comment:
What I like is this style of arguments (where the constructor parameter name is mentioned in the config): https://github.com/phpstan/phpstan-src/blob/990ba51cb38f29b17579c54c1967223fcc56661a/conf/config.neon#L327-L332

Reply:
Better indeed, applied and ✅